user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTab.java

```diff
@@ -100,7 +100,12 @@ public void createDeepLink() {
       UrlBuilder builder = new UrlBuilder();
       builder.setProtocol( Window.Location.getProtocol() );
       builder.setHost( Window.Location.getHostName() );
-      builder.setPort( Integer.parseInt( Window.Location.getPort() ) );
+      try {      
+        builder.setPort( Integer.parseInt( Window.Location.getPort() ) );
+      } catch (NumberFormatException e) {
```

Comment:
@SergeyTravin the formatting on this line doesn't match the pentaho standard, after that this is ready to merge.